frontend/components/songs/song_card_dropdown_item.jsx

```diff
@@ -13,29 +13,28 @@ const SongCardDropdownItem = ({
     removePlaylisted,
     createNewPlaylisted,
     createPlaylist,
-    displayPlaylist,
 }) => {
-    
     const runSongAction = (e) => {
         if (e.target.innerText === "Remove from this playlist") {
             updateSongCardDropdownState({ isOpen: false });
             return removePlaylisted(selectedSong.playlistedId);
-
         } else if (e.target.innerText === "Create new playlist") {
             updateSongCardDropdownState({ isOpen: false });
             const newPlaylist = {
                 title: `${selectedSong.title}`,
-                description: 'Please add a description',
-                user_id: currentUser.id,
+                description: "Please add a description",
+                user_id: currentUser.id, // Database/schema uses snake_case
             };
             createPlaylist(newPlaylist)
-                .then( (playlistId) => createNewPlaylisted(selectedSong.id, playlistId))
-                .then( (playlistId) => history.push(`/playlist/${playlistId}`));
+                .then((playlistId) =>
+                    createNewPlaylisted(selectedSong.id, playlistId)
+                )
+                .then((playlistId) => history.push(`/playlist/${playlistId}`));
         } else if (depthLevel === 1) {
             updateSongCardDropdownState({ isOpen: false });
             let playlist = playlists[index];
-            createNewPlaylisted(selectedSong.id, playlist.id)
-                .then( (playlistId) => history.push(`/playlist/${playlistId}`)
+            createNewPlaylisted(selectedSong.id, playlist.id).then(
```

Comment:
Is the `.then` supposed to be on the next line? Seems like a different style than above

Reply:
I used Prettier and it looks like it was moving what was inside .then to the next line only if it didn't fit character limit per line. This one fit so it kept it. I'll move it though for consistency.
```suggestion
            createNewPlaylisted(selectedSong.id, playlist.id)
            .then(
```